README.md

```diff
@@ -36,12 +36,12 @@ var el = {
 
 ## Usage
 
-### Converting to Types
+### Converting to Elements
```

Comment:
I feel like maybe this is more clear if we mention both the `to` and `from`:

Converting Javascript Values into Elements